packages/documentation/src/stories/components/alert.stories.tsx

```diff
@@ -0,0 +1,392 @@
+        type: 'boolean'
+      },
+      table: {
+        category: 'Icon',
+        type: {
+          summary: 'cssClass'
+        },
+        defaultValue: {
+          summary: 'no-icon'
+        }
+      }
+    },
+    dismissible: {
+      name: 'Dismissible',
+      description: 'Adds the dismissible styles.<br/><div className="text-danger">Do not forget to add the structural adjustments!</div>',
+      control: { type: 'boolean' },
+      table: {
+        category: 'Dismissible',
+        type: {
+          summary: 'cssClass'
+        },
+        defaultValue: {
+          summary: 'alert-dismissible'
+        }
+      }
+    },
+    fixed: {
+      name: 'Fixed',
+      description: 'Adds the fixed styles.<br/><div className="text-danger">Do not forget to add the structural adjustments!</div>',
+      control: { type: 'boolean' },
+      table: {
+        category: 'Fixed',
+        type: {
+          summary: 'cssClass'
+        },
+        defaultValue: {
+          summary: 'alert-fixed-bottom'
+        }
+      }
+    },
+    action: {
+      name: 'Action Buttons',
+      description: 'Adds the action button styles.<br/><div className="text-danger">Do not forget to add the structural adjustments!</div>',
+      control: { type: 'boolean' },
+      table: {
+        category: 'Action',
+        type: {
+          summary: 'cssClass'
+        },
+        defaultValue: {
+          summary: 'alert-action'
+        }
+      }
+    },
+    show: {
+      name: 'Show',
+      control: { type: 'boolean' },
+      table: { disable: true }
+    }
+  }
+} as Meta;
+
+function onShowToggle (e: React.MouseEvent, args: Args, updateArgs: Function) {
+  e.preventDefault();
+  updateArgs({ show: !args.show });
+}
+
+const Template = (args: Args) => {
+  const [_, updateArgs] = useArgs();
+
+  const classes = [
+    'alert',
+    args.variant,
+    args.icon !== 'null' ? `pi-${args.icon}` : '',
+    args.noIcon ? 'no-icon' : '',
+    args.dismissible ? 'alert-dismissible' : '',
+    args.fixed ? 'alert-fixed-bottom' : '',
+    args.action ? 'alert-action' : '',
+    args.show ? '' : 'd-none'
+  ].filter(c => c).join(' ');
+
+  // every time you want to parse one or more elements to a JSX.Element[], concat all the elements to a string and parse them all
+  // even if it would be possible to write some of them directly
+  // the reason for this is, that react elements in a list need a "key" property, single elements don't
+  // and the parser only creates this property, when it receives multiple elements to parse
+  const content: JSX.Element[] | JSX.Element | string = parse(`<h4 class="alert-heading" key="title">${args.title}</h4>${args.content}`);
+
+  return <div className={ classes } role="alert">
+    {/* Dismissible Button */}
+    { (args.dismissible || args.fixed) && !args.action ? <button className="btn-close" data-dismiss="alert" aria-label="Close" onClick={ (e: React.MouseEvent) => onShowToggle(e, args, updateArgs) }></button> : null }
```

Comment:
In the snippet, this function is replaced by `function noRefCheck(){}` which makes the code difficult to understand.

![image](https://user-images.githubusercontent.com/33580481/204845951-c6af9fd5-cbb0-45a3-8464-5f0edfd2d6d8.png)